df.creature-raws.xml

```diff
@@ -1250,8 +1250,8 @@
         <stl-vector name='tissue' pointer-type='tissue'/>
 
         <compound name='profession_name'>
-            <static-array type-name='stl-string' name='singular' count='135' index-enum='profession'/>
-            <static-array type-name='stl-string' name='plural' count='135' index-enum='profession'/>
```

Comment:
You updated this one for 0.44.11 already, right? (i.e. all of them should be `135`)

Reply:
I had updated a few already. But when I noticed that I had missed most of them I decided to try if automatic update would work.